### PR TITLE
Replace the Get Support button with Open Trader Chat until trade period is over

### LINKED ISSUE #3801: Emphasize that traders engage in trader chat to solve trade problems
To reduce number of mediation disputes we should emphazise in the UI that users are using the trade chat and engage together to solve the problem without the need of a mediator.
@m52go @ripcurlx Could you have a look how we could achieve that? Improving some of the popup text might be one way. To make trader chat more prominent another....

### PR DESCRIPTION
**Pull request by ghost**
Showing an `Open Trader Chat` button until the trade period is over will reduce the number of unnecessary support tickets and encourage trader-to-trader dialogue.  If the trade timer expires without completing the button changes (as before) to open a mediation ticket.

Implementation of this feature requires the button in TradeStepView to notify its parent TradeSubView which in turn notify its parent PendingTradesView that trader chat is to be opened.  Basically a callback interface is passed two levels down the GUI hierarchy.

Fixes #3801

### CODE REVIEW

**Comment by ghost** [comment]
example:

image

**Comment by ripcurlx** [comment]
@jmacxx I'm having a look at it right now. Please don't make any changes to translations except the English source file, as they are overwritten shortly before every release.

File: desktop/src/main/java/bisq/desktop/main/portfolio/pendingtrades/TradeSubView.java [hunk 1]
```diff
@@ -70,6 +72,9 @@ public TradeSubView(PendingTradesViewModel model) {
     }
 
     protected void activate() {
+
+
+
```
Comment (ripcurlx) [inline desktop/src/main/java/bisq/desktop/main/portfolio/pendingtrades/TradeSubView.java]: Is there a reason why you added this empty lines?

File: desktop/src/main/java/bisq/desktop/main/portfolio/pendingtrades/PendingTradesView.java [hunk 2]
```diff
@@ -260,6 +260,13 @@ protected void activate() {
                         root.getChildren().add(selectedSubView);
                     else if (root.getChildren().size() == 2)
                         root.getChildren().set(1, selectedSubView);
+
+                    // create and register a callback so we can be notified when the subview
+                    // wants to open the chat window
+                    ChatCallback chatCallback = trade -> {
+                        openChat(trade);
+                    };
```
Comment (ripcurlx) [inline desktop/src/main/java/bisq/desktop/main/portfolio/pendingtrades/PendingTradesView.java]:
```suggestion
                    ChatCallback chatCallback = this::openChat;
```

File: desktop/src/main/java/bisq/desktop/main/portfolio/pendingtrades/TradeSubView.java (hunk 1 above)
Comment (ripcurlx) [inline desktop/src/main/java/bisq/desktop/main/portfolio/pendingtrades/TradeSubView.java]:
```suggestion
```

File: desktop/src/main/java/bisq/desktop/main/portfolio/pendingtrades/steps/TradeStepView.java [hunk 3]
```diff
@@ -21,8 +21,10 @@
 import bisq.desktop.components.TitledGroupBg;
 import bisq.desktop.components.TxIdTextField;
 import bisq.desktop.main.overlays.popups.Popup;
+import bisq.desktop.main.portfolio.PortfolioView;
```
Comment (ripcurlx) [inline desktop/src/main/java/bisq/desktop/main/portfolio/pendingtrades/steps/TradeStepView.java]: Unused import
```suggestion
```

**Comment by ghost** [comment]
@ripcurlx  Ok, I have restored the language files to their original (de, es, pt, pt-br).

**Review by ripcurlx — CHANGES_REQUESTED** [review]
NACK - Please see my remarks

ACK functionality wise

File: desktop/src/main/java/bisq/desktop/main/portfolio/pendingtrades/TradeSubView.java (hunk 1 above)
Comment (ghost) [reply desktop/src/main/java/bisq/desktop/main/portfolio/pendingtrades/TradeSubView.java]: It was an accident.

**Review by ripcurlx — APPROVED** [review]
ACK

Tested it again on Regtest and everything works as expected.

Before trade period is over it will show

Bildschirmfoto 2020-04-10 um 16 15 30

and after the trade period it will show

Bildschirmfoto 2020-04-10 um 19 51 15

**Pull request merged**